app/views/staff/reports/_download.haml

```diff
@@ -1,2 +1,7 @@
-.govuk-grid-column-one-third.page-actions
-  = link_to t("action.report.download.button"), report_path(report_presenter, format: :csv), class: "govuk-link"
+.govuk-grid-column-two-thirds
+  %h3.govuk-heading-m
+    Download
+  %p.govuk-body
+    Download a CSV file to review offline. For guidance on reviewing and submitting your report, see the
+    = link_to_new_tab " guidance in the help centre", "https://beisodahelp.zendesk.com/hc/en-gb/articles/1500005604902-Review-and-Submit-your-Data"
+  = link_to t("action.report.download.button"), report_path(report_presenter, format: :csv), class: "govuk-button govuk-button--secondary"
```

Comment:
do you think we might be missing an opportunity to add to `scenario "can download a CSV of the report` some detail on the additional value being added here, e.g.

- I see a link to guidance about the reviewing my submission
- I see that the CSV is offered for review purposes

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/14feb522f29189a1a59ac4144c3d2a394df95659/spec/features/staff/users_can_view_reports_spec.rb#L143-L157

Reply:
Happy to. If I change the scenario to `they see helpful guidance about and can download a CSV of their own report` and check for the content and link, do you think that will do it?

Reply:
that sounds good: I think there's generally a strong case for trying to ensure that our tests describe the improvements we make: i.e. how does the change in behaviour deliver value?

Reply:
I don't disagree, I do wonder if there is a balance to strike at some point, but that is definately a conversation for later and I am confident you will convince me!